virt/storage/virt-configuring-storage-profile.adoc

```diff
@@ -8,9 +8,9 @@ toc::[]
 
 A storage profile provides recommended storage settings based on the associated storage class. A storage profile is allocated for each storage class.
 
-If the Containerized Data Importer (CDI) does not recognize your storage provider, you must configure storage profiles.
+The Containerized Data Importer (CDI) recognizes a storage provider if it has been configured to identify and interact with the storage provider's capabilities.
 
-For recognized storage types, CDI provides values that optimize the creation of PVCs.  However, you can configure automatic settings for a storage class if you customize the storage profile.
+For recognized storage types, the CDI provides values that optimize the creation of PVCs. You can configure automatic settings for a storage class if you customize the storage profile. If the CDI does not recognize your storage provider, you must configure storage profiles.
```

Comment:
You can configure automatic settings for a storage class if you customize the storage profile. If the CDI does not recognize your storage provider, you must configure storage profiles.  >>>>>>
In this case you may configure automatic settings for the storage class by customizing the storage profile.  In the case that CDI does not recognize your storage provider, you will have to configure the storage profiles.